Formula/xsimd.rb

```diff
@@ -0,0 +1,41 @@
+class Xsimd < Formula
+  desc "Modern, portable C++ wrappers for SIMD intrinsics"
+  homepage "https://xsimd.readthedocs.io/en/latest/"
+  url "https://github.com/QuantStack/xsimd/archive/6.1.5.tar.gz"
+  sha256 "ecc781520f7b2bfdeb51d1385683694f2e89be1a05b77b222b9783cfeb645034"
```

Comment:
Need a blank line after the `sha256` block.

Reply:
Thank you for the feedback. I've fixed the issue but am unsure whether or not to re-submit a new PR to adhere to the "One formula per commit; one commit per formula" rule.

Reply:
That's fine, we can always squash them into one commit later.

Reply:
OK that's nice to hear.